lldb/test/CMakeLists.txt

```diff
@@ -235,6 +235,8 @@ if (CMAKE_SIZEOF_VOID_P EQUAL 8)
   set(LLDB_IS_64_BITS 1)
 endif()
 
+set(LLDB_SHELL_TESTS_DISABLE_REMOTE OFF CACHE BOOL "Disable remote Shell tests execution")
```

Comment:
Remind me again what this does (probably the third time I've asked this, I keep forgetting :) ). When would I disable remote for this and what happens if I do?

Also should this be LLDB_TEST_SHELL_TESTS_DISABLE_REMOTE? So it sorts next to all the other ones?

Reply:
> Also should this be LLDB_TEST_SHELL_TESTS_DISABLE_REMOTE? So it sorts next to all the other ones?

Renamed it to LLDB_TEST_SHELL_DISABLE_REMOTE, so as not to duplicate TEST.


> Remind me again what this does (probably the third time I've asked this, I keep forgetting :) ). When would I disable remote for this and what happens if I do?

In case tests fail due to unsupported compiler/toolchain used, i.e. not clang, for building test executables. If set, this flag should cause Shell tests to be compiled and run against the local (host machine) target.

Reply:
Ok and that means you can still use a single `ninja check-lldb` command instead of having to split it up.

Seems like it will get limited use, but it's always good to have an escape hatch and this is not expensive to have.